sdk/v2_verificationMethod.go

```diff
@@ -1,4 +1,3 @@
-// DO NOT EDIT LOCAL SDK - USE v3 okta-sdk-golang FOR API CALLS THAT DO NOT EXIST IN LOCAL SDK
```

Comment:
😅

Reply:
"It's a bold strategy, Cotton. Let's see if it pays off for 'em"

Reply:
TL;DR: I think they're making changes to this file so they can expedite support for this feature ❤️ 

The old SDK is littered with these "do not edit" comments as the maintainers want to move away from maintaining a fork of the SDK
This is a good thing IMO as we should be upstreaming any patches rather than maintaining a bundled fork of the SDK

that said, these changes are pretty harmless IMO, they're just adding the missing type declarations to the old SDK so that it can support the new feature
I think the author did this to expedite support for this feature for you